ci/gem-push.yml

```diff
@@ -3,8 +3,6 @@ name: Ruby Gem
 on:
   push:
     branches: [ $default-branch ]
-  pull_request:
```

Comment:
I removed this because it is very unusual to publish unapproved contributions as a package from a CI workflow. This leads to cases where e.g a gem gets pushed to rubygems before its code is reviewed, and then even if it is approved, it gets pushed again when the PR is merged. An alternative would be to disable just the "Publish to RubyGems" step on PR.

Reply:
or, it may be better to use a release trigger [like in npm-publish](https://github.com/actions/starter-workflows/blob/main/ci/npm-publish.yml)

Reply:
… or in pypa/python publihsing. Yes, this should be updated to trigger on GH Release publish.

Reply:
cool, it uses a release trigger now.